### PR TITLE
A little shuffling of the outline

### PR DESCRIPTION
**Pull request by jhkennedy**
I've mostly just moved stuff around with regard to architecture and the plugins, which I've described in review comments below.

### CODE REVIEW

File: .github/dictionary.txt [hunk 1]
```diff
@@ -21,15 +26,10 @@ Geocoded
...
 HyP3
-HyP3-autorift
-HyP3-AutoRIFT
-HyP3-autoRIFT
 hyp3-autorift
-HyP3-cookiecutter
-HyP3-gamma
-HyP3-GAMMA
-HyP3-paper
-HyP3-SDK
```
Comment (jhkennedy) [inline .github/dictionary.txt:32]: Stylistically, I'd prefer:
* Names to look like "HyP3 Thing"
* repositories be written like "hyp3-thing"

and not do mixed-case hyphenation.

File: paper/outline.md [hunk 2]
```diff
@@ -34,66 +34,74 @@
...
+                    - Commercial software 
+                - HyP3 autoRIFT
+                    - Specialized algorithm finding pixel displacement between two radar or optical images developed by the ITS_LIVE team at JPL `[@Lei:2021]`
+            - custom (available in select custom deployments)
+                - DockerizedTopsApp
+                    - open source InSAR processing for Sentinel-1 Geocoded Unwrapped Interferograms (GUNW)
+                - Watermap (ASF Tools?)
+                    - Processing utilizing RTC products with different polarizations to estimate surface water extent and depth
+                - Riverwidth
+                    - processing utilizing RTC product with different polarization to estimate surface water extent and river widths
```
Comment (jhkennedy) [inline paper/outline.md:87]: I think it makes the most sense to talk about HyP3 as comprised of:
* the platform
* the plugins

So I'd like the Architecture section to have a sub-section for both. 

I'm not sure if we should only talk about the production (generally available) plugins here and move all the custom ones to a section about custom deployments, or just group them all together. That might fall out of the large writing effort.

Comment (jacquelynsmale) [reply paper/outline.md:87]: That makes sense to me. In that vein, do you think the HyP3 Access section would make sense before talking about the API Gateway or is it best at the end?

**Review by forrestfwilliams — APPROVED** [review]

File: paper/outline.md (hunk 2 above)
Comment (jhkennedy) [reply paper/outline.md:87]: I'm not sure. I suspect when writing it'll most naturally flow like:
* what it is
* why it is needed
* how to use it
* how we built it

But I the last two could go in the opposite order.  I think we'll find out as we write and right now the outline gets us moving in that direction so I'm happy with it :-D

**Pull request merged**